index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- Great job with this, @ceddddy :wink: 
- [ ] While this project might seem of little importance, please consider approaching it as a regular big project. It will be a lot better if you consider changing the current  `title` of your page from `document` to a more descriptive term that indicates the subject of the page so that your page has good Search Engine Optimization (SEO) ranking. This practice is also good for accessibility purposes as well as user experience. See how it looks at the moment:
![image](https://user-images.githubusercontent.com/99715640/211675188-957b0012-8bd2-47ba-addd-3cc3f4fca99a.png)